docs/reference/src/language/concepts/translations.md

```diff
@@ -195,13 +223,19 @@ int main()
 }
 ```
 
-Suppose you're using the above and the user's locale is set to `fr`,
-Slint will look for `my_application.mo` in the `lang/fr/LC_MESSAGES/` directory.
+For example, if you're using the above and the user's locale is `fr`,
+Slint looks for `my_application.mo` in the `lang/fr/LC_MESSAGES/` directory.
 
 ## Previewing Translations with `slint-viewer`
 
+<!-- TODO: Again, is this obvious? I see no instructions -->
+
 Use `slint-viewer` to preview translations when previewing `.slint` files:
 
+```rust
```

Comment:
Perhaps this could be in bullet point ("Ensure Slint-viewer is installed") or a `:::note` ?

(But feel free to leave it free-standing if you prefer)

Reply:
Adding into a note and doing the same for the extractor